ouroboros-consensus/src/Ouroboros/Consensus/Node.hs

```diff
@@ -145,6 +146,12 @@ data RunNodeArgs blk = RunNodeArgs {
       --
       -- Use 'defaultClockSkew' when unsure.
     , rnMaxClockSkew :: ClockSkew
+
+      -- | Maximum concurrent downloads during bulk syncing.
+    , rnMaxConcurrencyBulkSyn :: Word
+
+      -- | Maximum requests in flight during bulk syncing.
+    , rnMaxConcurrencyDeadline :: Word
```

Comment:
What about `rnCustomMiniprotocolParameters :: MiniProtocolParameters -> MiniProtocolParameters`. This approach allows us to define the defaults in this module instead of deferring that responsibility to `cardano-node` while allowing them to be overridden in `cardano-node`.

Reply:
`MiniProtocolParameters` represents hard limits and are not really configurable. I'm not touching them in this PR.

Reply:
Ah, I just realised that `BlockBlockFetchConfiguration` is not a subset of `MiniProtocolParameters`.